testproxy/known_failures.txt

```diff
@@ -1,9 +1,9 @@
 TestMutateRow_Generic_DeadlineExceeded\|
+TestMutateRow_Generic_CloseClient\|
```

Comment:
mutateRow service isn't using the veneer so this test passing before was a false positive.